lib/util/cache/global/index.ts

```diff
@@ -1,10 +1,16 @@
 import * as runCache from '../run';
+import * as fileCache from './file';
+
+let initComplete = false;
 
 function getGlobalKey(namespace: string, key: string): string {
   return `global%%${namespace}%%${key}`;
 }
 
 export function get<T = any>(namespace: string, key: string): Promise<T> {
+  if (!initComplete) {
```

Comment:
Simply test for `global.renovateCache` here, so you are able to override it in the app

Reply:
True, that's a better stepping stone. I plan to add Redis support here after this

Reply:
What about initialize `renovateCache` in this file init function and simply set `global.renovateCache = fileCache` ?